api/controllers/project.js

```diff
@@ -988,6 +988,8 @@ exports.protectedPublish = function (args, res, next) {
   Project.findOne({ _id: objId }, function (err, o) {
     if (o) {
       defaultLog.info("o:", o);
+      o.currentLegislationYear = "legislation_" + args.body.legislationYear;
```

Comment:
might want to do a null check on this. And also how is the legislation list logic handled?

Reply:
we will handle the legislation list logic on the put